pkg/comp-functions/functions/common/password.go

```diff
@@ -74,3 +80,26 @@ func genPassword() (string, error) {
 
 	return gen.Generate(16, 4, 0, false, true)
 }
+
+func AddStaticFieldToSecret(staticFields map[string]string) CredentialSecretOption {
```

Comment:
Please add a comment what this function does. As it's part of the public API of the common package, it's important that it's documented.

Reply:
Done